File: sem 2/C++/lista4/1.cpp
#include <iostream>
#include <vector>

#include "mylib.hpp"

using namespace std;

int main()
{
	int num;
	cout<<"N: "; cin>>num;
	
	vector <int> tab;
	for(int n = 0, a; n < num; n++)
	{
		cin>>a;
		tab.push_back(a);
	}
	
	int tmp = my::min(tab.begin(), tab.size());
	cout<<"Min: "<<tab[tmp]<<" at "<<tmp<<endl;
	tmp = my::max(tab.begin(), tab.size());
	cout<<"Max: "<<tab[tmp]<<" at "<<tmp<<endl;
	cout<<"Find: ";
	cin>>tmp;
	cout<<"Index: "<<my::find(tab.begin(), tab.size(), tmp)<<endl;
}

File: sem 3/python/2/1.py
import mylib

numbers = []
while len(numbers) < 15:
	numbers.extend(input("Podaj {} liczb: ".format(15-len(numbers))).split(" "))
numbers = [int(x) for x in numbers]

print("szukana: ", mylib.find(numbers, int(input("Podaj liczbe do znalezienia: "))))
i = mylib.min(numbers)
print("min index: ", i)
print("min: ", numbers[i])
i = mylib.max(numbers)
print("max index: ", i)
print("max: ", numbers[i])
print("srednia: ", mylib.avg(numbers))
print("srednia geometryczna: ", mylib.avg2(numbers))